tests/Controllers/ScorePinsControllerTest.php

```diff
@@ -79,7 +79,7 @@ public function testReorderMoveBottom()
         $user = User::factory()->create();
         $pins = collect([0, 1])->map(fn ($order) => ScorePin
             ::factory(['display_order' => $order])
-            ->withScore(static::createScore($user, Ruleset::osu))
+            ->withScore(static::createScore($user, Beatmap::MODES['osu']))
```

Comment:
I'm not sure if this is any better (or worse 🤷 ) than using `Ruleset` in this case